tests/unit/test_routes/test_invalid_put_post.py

```diff
@@ -97,15 +97,15 @@ def test_validate_resource(module_client, module_db, fake_auth_from_oc):
                           )
     assert_missing_body(response)
 
-    # Data cannot be empty
+    # Data must be valid json
```

Comment:
This test was not quite working as intended previously. Passing in `data=''` is not the same as not passing in `data` at all. This ultimately gets treated as a 400 bad request as there is a payload, but it's not valid json.